client/src/containers/navbar/index.js

```diff
@@ -34,16 +40,19 @@ import NavigationExpandMoreIcon from 'material-ui/svg-icons/navigation/menu'
 */
 export default () => (
   <header>
-    <Toolbar>
+    <Toolbar style={{ backgroundColor: "#ea4e46" }}>
```

Comment:
I think we're trying to avoid inline styling. For the MUI components, see the link Halala posted about changing MUI themes, or see her example she used for the profile edit view: https://github.com/iandusenbury/tableland/blob/f352bf556da0bfbedb44944759fa551ed0e03b15/client/src/index.js

Reply:
I followed this suggestion and moved the pink background color, as well as all of the MESA colors from the MESA Styles document to the "src/index.js" file